### PR TITLE
add style draw in yolo

### PR DESCRIPTION
**Pull request by cabelo**
### This pullrequest changes 

Added style in the Yolo drawing.

Example below:

### CODE REVIEW

**Comment by cabelo** [comment]
Example :
example_style

File: samples/dnn/yolo_object_detection.cpp [hunk 1]
```diff
@@ -156,8 +164,9 @@ int main(int argc, char** argv)
                     int baseLine = 0;
                     Size labelSize = getTextSize(label, FONT_HERSHEY_SIMPLEX, 0.5, 1, &baseLine);
                     rectangle(frame, Rect(Point(xLeftBottom, yLeftBottom ),
-                                          Size(labelSize.width, labelSize.height + baseLine)),
-                              Scalar(255, 255, 255), CV_FILLED);
+                              Size(labelSize.width, labelSize.height + baseLine)),
```
Comment (alalek) [inline samples/dnn/yolo_object_detection.cpp]: This indentation change does not correct: `Size()` is argument of `Rect()` ctor, not `rectangle()` call.

Comment (cabelo) [reply samples/dnn/yolo_object_detection.cpp]: Okay, but the code was already in the previous version.

I will fix

Comment (cabelo) [reply samples/dnn/yolo_object_detection.cpp]: ok, I understand sorry

Comment (cabelo) [reply samples/dnn/yolo_object_detection.cpp]: done

**Comment by dkurt** [comment]
@cabelo, do we really need it? It's hard to understand an accuracy of predictions this way. I mean the same line might be drawn as for good bounding box as well for a poor one.
screenshot from 2017-12-28 12-35-19

**Comment by cabelo** [comment]
@dkurt  I think that sometimes the boxes pollute the image a lot to analyze.

But you decide what's best.

**Comment by alalek** [comment]
Rectangle diagonal line is enough to restore the original rectangle (with lines parallel X/Y axis), so I believe this line may be enough.

**Comment by dkurt** [comment]
@alalek, or corners.
screenshot from 2017-12-28 17-04-16

**Comment by alalek** [comment]
@dkurt Right.

BTW, In general, this is the sample, not a research tool. We should keep samples as simple as possible, just enough to demonstrate OpenCV functionality.

**Comment by cabelo** [comment]
I'll make corners ok?

**Comment by alalek** [comment]
I believe the current state of this patch is fine.

**Comment by cabelo** [comment]
Ok @alalek  thank you.

Until the next patch

Bye

**Comment by alalek** [comment]
I have pushed minor code refactoring update (see commit comments) including `cv::format` changes from #10483

**Comment by dkurt** [comment]
:+1:

**Pull request merged**